admin_manual/release_notes.rst

```diff
@@ -74,13 +74,17 @@ The Password Policy Extension
 .. note::
    These features don't apply to users imported from LDAP or other backends but only for local users created by administrators or the `Guests <https://marketplace.owncloud.com/apps/guests>`_ extension.
 
-Imposing password expiration and history policies enhance security. For example, when users are forced to choose a new password, they can be blocked from using their previous passwords (the password history count is configurable). However, increasing security can also decrease user satisfaction, such as forcing them to change their password upon first login, or change their passwords too frequently (such as every 90 days). That said, we encourage an acceptable balance between security and usability.
+Imposing password expiration and history policies enhances security. For example, when users are forced to choose a new password, they can be prevented from using their previous passwords. It is possible to manually expire passwords or to configure the number of days that have to pass since the last change before the password expires. To ensure a good experience it is possible to configure user information via email and internal notifications that appear on the web interface and clients. The password history count and the days before the reminder notification is sent are both configurable. Users will always be informed when passwords have expired.
+However, increasing security can also decrease user satisfaction, such as forcing them to change their password upon first login, or change their passwords too frequently like every 30 days, for example. That said, we encourage an acceptable balance between security and usability by choosing sensible settings for password policies. A high frequency of password changes might increase security but could also decrease user satisfaction.
```

Comment:
you have repeated "user satisfaction". in my comment I meant to remove this part.
Maybe then just remove the last sentence ?

Reply:
Changed, thx for spotting.